### PR TITLE
support JVMChaos in Dashboard

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
Signed-off-by: xiang <xiang13225080@163.com>

### What problem does this PR solve?

> Problem Summary:
support JVMChaos in Dashboard 

### What's changed and how it works?

> What's Changed:

### Related changes

- [x] Need to update `chaos-mesh/website`: https://github.com/chaos-mesh/website/pull/165
- [ ] Need to update `Chaos Dashboard`
- [ ] Need to **cheery-pick to release branches**

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Breaking backward compatibility

### Release note 

```text
Please add a release note.

You can safely ignore this section if you don't think this PR needs a release note.
```

### CODE REVIEW

**Review by g1eny0ung — APPROVED** [review]
LGTM 🍻

File: ui/src/components/NewExperimentNext/data/types.ts [hunk 1]
```diff
@@ -144,6 +144,30 @@ const ioCommon: Spec = {
...
+    value: '',
+    helperText: 'Method that is injected with faults',
+  },
+}
+
+const jvmPort: Spec = {
+  port: {
+    field: 'number',
+    label: 'Port',
+    value: 9288,
```
Comment (STRRL) [inline ui/src/components/NewExperimentNext/data/types.ts:166]: I prefer to not set this default value, or keep consistent with `DefaultJVMAgentPort`, `9277`.

rest LGTM!

Comment (STRRL) [reply ui/src/components/NewExperimentNext/data/types.ts:166]: PTAL @WangXiangUSTC

Comment (WangXiangUSTC) [reply ui/src/components/NewExperimentNext/data/types.ts]: I update it to 9277

Comment (WangXiangUSTC) [reply ui/src/components/NewExperimentNext/data/types.ts]: I don't think users should care about this configuration unless it uses the same port number, so give it a default value

**Review by STRRL — APPROVED** [review]
LGTM!

**Pull request merged**